nodes:
  - name: scan_raw
    topic: /scan_raw
    msg: sensor_msgs::msg::LaserScan
    node_type: Producer  
    qos_profile:
      qos_history: KEEP_LAST
      qos_queue: 100
      qos_reliability: RELIABLE
      qos_durability: VOLATILE
    topic_name: my_scan
  - name: nav_vel
    topic: /nav_vel
    msg: geometry_msgs::msg::Twist
    node_type: Consumer
    qos_profile:
      qos_history: KEEP_ALL
      qos_queue: 10
      qos_reliability: BEST_EFFORT
      qos_durability: TRANSIENT_LOCAL
  - name: image_raw
    topic: /head_front_camera/rgb/image_raw
    msg: sensor_msgs::msg::Image
    node_type: Producer
